consul/metadata.csv

```diff
@@ -7,3 +7,33 @@ consul.catalog.services_critical,gauge,,service,,Total critical services on node
 consul.catalog.services_passing,gauge,,service,,Total passing services on nodes,1,consul,svc pass
 consul.catalog.services_up,gauge,,service,,Total services registered on nodes,0,consul,svc up
 consul.catalog.services_warning,gauge,,service,,Total warning services on nodes,-1,consul,svc warn
+consul.net.node.latency.min,gauge,,millisecond,,minimum latency from this node to all others,-1,consul,min latency
+consul.net.node.latency.p25,gauge,,millisecond,,p25 latency from this node to all others,-1,consul,p25 latency
+consul.net.node.latency.median,gauge,,millisecond,,median latency from this node to all others,-1,consul,median latency
+consul.net.node.latency.p75,gauge,,millisecond,,p75 latency from this node to all others,-1,consul,p75 latency
+consul.net.node.latency.p90,gauge,,millisecond,,p90 latency from this node to all others,-1,consul,p90 latency
+consul.net.node.latency.p95,gauge,,millisecond,,p95 latency from this node to all others,-1,consul,p95 latency
+consul.net.node.latency.p99,gauge,,millisecond,,p99 latency from this node to all others,-1,consul,p99 latency
+consul.net.node.latency.max,gauge,,millisecond,,maximum latency from this node to all others,-1,consul,max latency
+consul.runtime.num_goroutines,gauge,10,,,Number of running goroutines,0,consul,num goroutines
+consul.runtime.alloc_bytes,gauge,10,byte,,Current bytes allocated by the Consul process,0,consul,bytes alloc
+consul.runtime.heap_objects,gauge,10,object,,Number of objects allocated on the heap,0,consul,heap objs
+consul.runtime.sys_bytes,gauge,10,byte,,Total size of the virtual address space reserved by the Go runtime,0,consul,
```

Comment:
I have no objections to leaving them blank if we can't come up with a decent short version